events/wc_frostwolf_events.txt

```diff
@@ -1086,34 +1127,293 @@ narrative_event = { #Frostwolf settled
+	}
+	
+	picture = GFX_evt_orcish_horde
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+	hide_from = yes
+	
+	option = {
+		trigger = { character = event_target:target_frostwolf_leader }
+		name = EVTOPT_WCFW_62 # May the spirits curse you forever, [From.GetFirstName]!
+	}
+	
+	option = {
+		trigger = { NOT = { character = event_target:target_frostwolf_leader } }
+		name = INTERESTING
+	}
+	
+	after = {
+		# Create landless title for new orc (so he can be landless)
+		create_title = {
+			tier = DUKE
+			landless = yes
+			holder = event_target:target_new_orc
+			culture = event_target:target_frostwolf_butcher
+		}
+		
+		if = {
+			limit = { character = event_target:target_frostwolf_leader }
+			# Become adventurer orc if you're Frostwolf leader, they have a landless title so no game over
+			set_player_character = event_target:target_new_orc
+		}
+		
+		# Now butcher the whole frostwolf clan, abdicate to new orc + kill leader, killer unknown since done 'in secret'
+		event_target:target_frostwolf_leader = {
+			any_courtier = {
+				death = { death_reason = death_murder_unknown killer = event_target:target_frostwolf_butcher }
+			}
+			
+			# Destroy the Frostwolf tribe title and give the rest to the new orc
+			destroy_landed_title = d_frostwolf
+			abdicate_to = event_target:target_new_orc
+			death = { death_reason = death_murder_unknown killer = event_target:target_frostwolf_butcher }
+		}
+		
+		# Destroy the landless title so Dead Ravine becomes primary + make loyal vassal of Gul'dan/successor
+		event_target:target_new_orc = {
+			primary_title = {
+				activate_title = { title = THIS status = no }
+				destroy_landed_title = THIS
+			}
+			set_defacto_liege = event_target:target_frostwolf_butcher
+			opinion = { modifier = opinion_loyal_servant who = event_target:target_frostwolf_butcher }
+		}
+		
+		# Fire the event that speaks of the new inhabitants after three days (news has to travel, you know!)
+		73 = {
+			province_event = {
+				id = WCFW.163
+				days = 3
+			}
+		}
+	}
+}
+
+# delayer event
+province_event = {
+	id = WCFW.163
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	immediate = {
+		any_player = {
+			limit = {
+				realm = { title = e_horde }
+			}	
+			narrative_event = { id = WCFW.63 }
+		}
+	}
+}
+
+# NEW INHABITANTS
+# Delayed event
+narrative_event = {
+	id = WCFW.63
+	title = EVTTITLE_WCFW_63
+	desc =  EVTDESC_WCFW_63 # The decimation of the Frostwolf clan left their lands uninhabited...
+	picture = GFX_evt_orc_city
+	border = GFX_event_narrative_frame_war
+
+	is_triggered_only = yes
+	
+	immediate = {
+		d_stormreaver = { # Gul'dan or successor
+			holder_scope = {
+				save_event_target_as = target_frostwolf_butcher
+			}
+		}
+		c_dead_ravine = { # New orc (this early in the game, they're always here)
+		holder_scope = {
+				save_event_target_as = target_new_orc
+			}
+		}
+	}
```

Comment:
These scopes are already defined in the event chain earlier. Why they're here?

Reply:
Because you've taught me that CK2 loves to fuck shit up and that I should always expect the unexpected.

`WCFW.62` saves these event targets but `WCFW.63` triggers 3 days after that.. Either of them could have died already (although granted if the new one did, it'd be game over) but still.. You always told me to sorta plan ahead for that.

I can change it though if you'd like?

Reply:
That's why it's strange that you save holder of `c_dead_ravine` as "new orc". The Frostwolves may get other title from Warchief.
I wouldn't show this event if Gul'dan and "new orc" died. It's just notification.

Reply:
Hrrrh, I'll look into this.

Reply:
Would a trigger in `WCFW.163` (the delayer of this one) like the following suffice?
```
trigger = {
  AND = {
    event_target:target_frostwolf_butcher = {
      is_alive = yes
    }
    event_target:target_new_orc = {
      is_alive = yes
    }
  }
}
```

Or are there better/easier ways to write it?

Reply:
This should be changed in the latest commit.